components/bl-rating-component/src/helpers/use-rating-class-list.js

```diff
@@ -0,0 +1,11 @@
+import { useMemo } from 'react';
+
+export const useRatingClassList = (disabled, classList) => useMemo(() => {
```

Comment:
Could we change the order?

`classList, disabled`

The idea is first argument must be most important